server/src/configs/default.json

```diff
@@ -23,7 +23,8 @@
       "quests": false,
       "raids": false,
       "nests": false
-    }
+    },
+    "weatherHistoryDays": 3
```

Comment:
:feelslapras: Weather History _Days_????

Reply:
😅 couldn't think of a better key name. Thank you for the reminder though to add to the PR notes!

Reply:
🤔 Hmm good point, not really sure what you would call it, maybe  `weatherCellScanTimeLimit` possibly or sometime similar? Just throwing random names out here.

Reply:
Probably makes sense to have `limit` in there somewhere at the very least.